### PR TITLE
[1.x] Return http response instead of body

### PR DESCRIPTION
**Pull request by eiabea**
In order to do some kind of error handling or status code checking i propose to return the whole `Response` object instead of just the body

We had some issues with our `auth key` and would love to do something like this:

```
$response = Larafirebase::withTitle('Test Push from laravel command')
            ->withBody('Fingers crossed the device is getting this notification!')
            ->sendMessage($fcm);

if($response->statusCode() == 401){
  $this->info('Your auth key seems to be invalid');
}
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+vendor
```
Comment (gentritabazi01) [inline .gitignore:1]: Could you remove this for now?

File: src/Services/Larafirebase.php [hunk 2]
```diff
@@ -72,7 +73,7 @@ public function withSound($sound)
 
         return $this;
     }
-     
+
```
Comment (gentritabazi01) [inline src/Services/Larafirebase.php:76]: Good catch.

File: .gitignore (hunk 1 above)
Comment (eiabea) [reply .gitignore]: Removed in latest commit

File: src/Services/Larafirebase.php (hunk 2 above)
Comment (eiabea) [reply src/Services/Larafirebase.php:76]: Linting FTW! :smile:

**Comment by gentritabazi01** [comment]
Would be good to have documentation on Readme.

**Review by gentritabazi01 — APPROVED** [review]

**Pull request merged**